api/src/lib/cart/src/delete-cart.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @param {cartPgRepo} cartRepo
+ */
+function DeletecartFactory(deps) {
+    'use strict'
+
+    const { cartRepo } = deps
+
+    const deleteCart = async (ctx) => {
```

Comment:
#picky: I think this deletes an item from the cart, rather than a whole cart, so a more expressive name might be, `deleteProductFromCart`